### DISCUSSION TITLE
How can I show my Github profile and username on another site?

### DISCUSSION THREAD

**Discussion by bark1n0** [community/community#27884] [Other Feature Feedback, Questions, & Ideas]
How can I show my Github profile and username on another site (when I change my username, it will change on the site)

**Comment by EXELANCE-LLC** [comment]
You can do whatever you want by converting your profile page to html embed code.

**Comment by jfullstackdev** [comment]
> username on another site (when I change my username, it will change on the site)

maybe you mean, not your username as it is your ID, it will result to unintended side-effects, you can change it but not usually

maybe it's embedding that you are referring to, for example, if you create a tutorial on Gist, then there is the embed code, simply paste it on another site, if you update that gist, it will update on that site too
